### PR TITLE
feat: create new Ansible endpoint type

### PR DESCRIPTION
**Pull request by vkrizan**
Adds Event-Driven Ansible type that aliases WebHook processor.

RHCLOUD-25405

### CODE REVIEW

File: common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java [hunk 1]
```diff
@@ -10,7 +10,9 @@ public enum EndpointType {
     @JsonProperty("email_subscription")
     EMAIL_SUBSCRIPTION(false),
     @JsonProperty("camel")
-    CAMEL(true);
+    CAMEL(true),
+    @JsonProperty("eda")
```
Comment (vkrizan) [inline common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java:14]: @jeromemarc internally the type is gonna be "eda". Do we want to use a longer name?

**Comment by josejulio** [comment]
We also need to update this file https://github.com/RedHatInsights/notifications-backend/blob/f439ed0704386525c15fbb12d2fd365ac17b8f1e/common/src/main/java/com/redhat/cloud/notifications/models/Endpoint.java#L85

something like:

```
 @JsonSubTypes.Type(value = WebhookProperties.class, name = "eda"), 
```

File: common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java (hunk 1 above)
Comment (vkrizan) [reply common/src/main/java/com/redhat/cloud/notifications/models/EndpointType.java]: I've decided to go with "ansible". It's better than an abbreviated EDA, and the abbreviation can likely change.

**Comment by gwenneg** [comment]
@vkrizan When we met last week, my mind was focused on the Kafka messages processing and I forgot that we needed to add the `ANSIBLE` endpoint type at several places to properly create/update/load the properties for that endpoint type. Everything should be good now.

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by gwenneg** [comment]
The PR check failure is unrelated.

**Pull request merged**